src/Compatibility/Microsoft.DotNet.ApiCompatibility/Rules/EnumValuesMustMatch.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+
+namespace Microsoft.DotNet.ApiCompatibility.Rules
+{
+    public class EnumValuesMustMatch : Rule
```

Comment:
I know perhaps old rules didn't do this, but it would be good if we add tripple slash comments on all of these rules with a description and remarks on what each is checking.

Reply:
Filed https://github.com/dotnet/sdk/issues/25565.